code/modules/cargo/packs/misc.dm

```diff
@@ -162,7 +162,7 @@
 
 /datum/supply_pack/misc/lewd
 	name = "Lewd Crate" // OwO
-	desc = "Psss want to have a good time with your sluts? Well I got what you want maid clothing, dildos, collars and more!"
+	desc = "Pssst, want to have a good time with your sluts? Well I got what you want; maid clothing, dildos, collars and more!"
```

Comment:
probably should be a colon instead of a semicolon (sorry)

Reply:
Was going off the basis that semicolons are used for at the beginning of lists of things in sentences, but fair enough